.github/actions/setup-arc-e2e/action.yaml

```diff
@@ -0,0 +1,55 @@
+name: 'Setup ARC E2E Test Action'
+description: 'Build controller image, create kind cluster, load the image, and exchange ARC configure token.'
+
+inputs:
+  github-app-id:
+    description: 'GitHub App Id for exchange access token'
+    required: true
+  github-app-pk:
+    description: "GitHub App private key for exchange access token"
+    required: true
+  github-app-org:
+    description: 'The organization the GitHub App has installed on'
+    required: true
+  docker-image-name:
+    description: "Local docker image name for building"
+    required: true
+  docker-image-tag:
+    description: "Tag of ARC Docker image for building"
+    required: true
+
+outputs:
+  token:
+    description: 'Token to use for configure ARC'
+    value: ${{steps.config-token.outputs.token}}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+      with:
+        version: latest
```

Comment:
Should we pin the action version or buildx version?

Reply:
Should we be aware of if the latest is broken?
We don't restrict the `buildx` version in the makefile, why do we need to restrict it in the workflow?

Reply:
I mean for makefile, it will just use the buildx version you have installed on your system right?
and as long as we are able to build our image with some specific version and run it, it should not matter if the latest buildx version is breaking our build
It is like, if the go build breaks on the next go version, we would not care about it until we bump the version up:
https://github.com/actions/actions-runner-controller/blob/bd9f32e3540663360cf47f04acad26e6010f772e/.github/workflows/validate-arc.yaml#L37

Reply:
Ok, I am using the same version as `publish-runner-scale-set.yaml` now.